tests/legacy-cli/e2e/assets/14.0-project/src/app/app.component.html

```diff
@@ -312,9 +312,8 @@
   <span>Welcome</span>
     <div class="spacer"></div>
     <a aria-label="Angular on twitter" target="_blank" rel="noopener" href="https://twitter.com/angular" title="Twitter">
```

Comment:
You also need to change the title and aria label from `Twitter` to `X`

Reply:
Ohh My bad, I have done required changes. Please check once again